images/router/haproxy/conf/haproxy-config.template

```diff
@@ -423,6 +429,7 @@ backend be_secure:{{$cfgIdx}}
 
 # Secure backend, pass through
 backend be_tcp:{{$cfgIdx}}
+  id {{ .InternalID }}
```

Comment:
This `id {{ .InternalID }}` is just for passthrough backend right? but seems it is not appended to `server {{$endpoint.ID}} {{$endpoint.IP}}:{{$endpoint.Port}} weight {{$weight}}` (Line 471)